<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_thunderclassroom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0c1b71;&quot;&gt;THUnder Classroom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>301</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_username">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_username"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_pswd">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_pswd"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_ip"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_serverip">
       <property name="text">
        <string>Server IP:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_pswdalert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
